src/controllers/payments.js

```diff
@@ -140,9 +141,11 @@ exports.put = function *(id) {
 
   payment.id = id.toLowerCase()
 
-  let isPrepared = yield Payments.validate(payment)
-  if (isPrepared) {
-    yield Payments.settle(payment)
+  yield Payments.validate(payment)
+  const truePayment = settlementQueue.storePayment(payment)
```

Comment:
The operations belong together. I agree that `store` isn't the most descriptive verb for this, but the doc and code make it clear whats going on, so I don't think its an issue.